<?php

namespace Container8DQZO9V;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class get_ServiceLocator_CDhANYIService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.CDhANYI' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.CDhANYI'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService, [
            'entityManager' => ['services', 'doctrine.orm.default_entity_manager', 'getDoctrine_Orm_DefaultEntityManagerService', true],
            'tache' => ['privates', '.errored..service_locator.CDhANYI.App\\Entity\\Tache', NULL, 'Cannot autowire service ".service_locator.CDhANYI": it references class "App\\Entity\\Tache" but no such service exists.'],
        ], [
            'entityManager' => '?',
            'tache' => 'App\\Entity\\Tache',
        ]);
    }
}
